BusBoard.ConsoleApp/TflApi.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using RestSharp;
+
+namespace BusBoard.ConsoleApp
+{
+    public class TflApi
+    {
+        public IEnumerable<BusStop> GetTwoClosestBusStopsToPostcode(string postcodeString)
+        {
+            var requestUrl = "http://api.postcodes.io";
+            var client = new RestClient(requestUrl);
+            var request = new RestRequest($"postcodes/{postcodeString}", DataFormat.Json);
+            var response = client.Execute(request);
+            var postcodeApiResponse = JsonConvert.DeserializeObject<PostcodeApiResponse>(response.Content);
+            var postcode = postcodeApiResponse.Result;
+            List<BusStop> nearbyStops = GetBusStopsNearPostcode(postcode);
+            IEnumerable<BusStop> closestTwoStops = nearbyStops.OrderBy(stop => stop.Distance).Take(2);
+            return closestTwoStops;
```

Comment:
Very minor, but I'd probably have just called `return` on line 19 without creating this variable as it's not used anywhere else. And we already know its type because it's in the method signature.

Reply:
yeah sure